analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs

```diff
@@ -147,7 +147,7 @@ public static class OldVerifier
                 .WithConcurrentAnalysis(false)
                 .WithOptions(ParseOptionsHelper.CSharpPreview)
                 .Verify();
-
+        //FIXME: Remove after rebase
```

Comment:
This overload will be removed after #5293 gets merged and this PR rebased

Reply:
#5293 is merged, so you can rebase and fix the part that is begging with a FIXME :)